src/sql/workbench/contrib/dashboard/browser/widgets/explorer/explorerWidget.component.html

```diff
@@ -7,6 +7,8 @@
 <div class="explorer-widget" style="display: flex; flex-flow: column; position: absolute; height:100%; width:100%; padding: 10px; box-sizing: border-box">
 	<div #input style="width: 100%"></div>
 	<div style="flex: 1 1 auto; position: relative">
+		<loading-spinner [loading]="loading" [loadingMessage]="loadingMessage"
```

Comment:
At some point we should probably move over to using the standard progress indicator VS Code uses (same type as the portal as well)

![Capture](https://user-images.githubusercontent.com/28519865/77330012-fc9ce600-6cdb-11ea-8bb6-cd982a3617e2.gif)

https://github.com/microsoft/azuredatastudio/blob/maste/src/vs/platform/progress/common/progress.ts#L16

Reply:
interesting, this definitely makes sense for search/filter scenario, but for the first time load/refresh scenario, I personally prefer the spinner. lets consult the ux team about this.

if you are talking about the filtering scenario for explorer widget, we probably don't need it since the filtering is done in memory and it almost finishes immediately.

Reply:
Yeah it's definitely worth discussing when we have multiple components on a page that are loading separately. In the portal they use the triple dot loading thing per element so that's basically the same. But when we have a whole page loading then it'd probably make sense to just disable the page and use something like this. 

Aren't you adding the spinner here though? Because yeah I agree it seems a bit odd to have the spinner when it's basically never going to actually show.

Reply:
Oh that's for the initial load. Yeah that makes sense.